tests/openwisp2/tests/user_batch_test.py

```diff
@@ -0,0 +1,317 @@
+import os
+
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.urls import reverse
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import Select, WebDriverWait
+
+from .test_selenium import SeleniumTestMixin
+
+current_script_path = os.path.dirname(os.path.abspath(__file__))
+
+
+class BasicTest(SeleniumTestMixin, StaticLiveServerTestCase):
+    def setUp(self):
+        self.admin = self._create_admin(
+            username=self.admin_username, password=self.admin_password
+        )
```

Comment:
This is already done by the `setUp` method of `openwisp_utils.tests.SeleniumTestMixin`